protocol/answer/answer.json

```diff
@@ -21,6 +21,46 @@
                     },
                     "minItems": 0,
                     "$comment": "Return value for a `catchup` request"
+                },
+                {
```

Comment:
you can absolutely define this whole block in a separate file and use a "$ref" to keep things concise.

Reply:
I added a file `messages_by_channel.json` to address this.